src/camera.js

```diff
@@ -649,6 +655,14 @@
 
     /**
      * Public exposure of the viewFromCenterSizeRotation function.
+     * @param {object} center
+     * @param {number} center.x
+     * @param {number} center.y
```

Comment:
should we make a note of units for size (pixels) and center?

Reply:
Also, the private function has the explanation of what the function does but not the public function. The API docs generated shows only one line. thoughts?